library/pom.xml

```diff
@@ -308,6 +308,16 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+          <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
What is the purpose of this change? If this is unintensional, please revert this file. As such we should be very careful in modifying pom.xml as this might break things at other places.